File: core/j-SDK-core/src/main/java/jOS/Core/SDKChangelogActivity.java
package jOS.Core;

import android.os.Bundle;

public class SDKChangelogActivity extends jWebActivity{

    @Override
    public void onCreate(Bundle savedInstanceState) {
        configure("https://github.com/dot166/jOS_j-sdk/commits/v" + BuildConfig.SDKVersion, true, false, true, false, true);
        super.onCreate(savedInstanceState);
    }
}

File: core/j-SDK-core/src/main/java/jOS/Core/SDKChangelogActivity.kt
package jOS.Core

import android.os.Bundle

class SDKChangelogActivity : jWebActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        configure("https://dot166.github.io/jOS/commlib/Changelog/", true, false, true, false, true)
        super.onCreate(savedInstanceState)
    }
}
